ts/src/base/Exchange.ts

```diff
@@ -2304,7 +2304,11 @@ export default class Exchange {
          * @description safely extract boolean value from dictionary or list
          * @returns {bool | undefined}
          */
-        return this.safeBoolN (dictionary, [ key ], defaultValue);
+        const value = this.safeValue (dictionary, key, defaultValue);
+        if (typeof value === 'boolean') {
```

Comment:
@ttodua insteadof typeof value isn't faster to do value === true or value === false

Reply:
ahh, seems this breaks python https://github.com/ccxt/ccxt/actions/runs/20442073697/job/58737144955?pr=27375#step:14:23
wdyt?